var/spack/repos/builtin/packages/fox/package.py

```diff
@@ -19,12 +19,9 @@ class Fox(AutotoolsPackage):
 
     license("LGPL-3.0-or-later")
 
+    version("1.7.84", sha256="bdb1fe785605488b58addc95f6091a75873e8a3bea7b83caecfb7f4b0827b34e")
```

Comment:
fox toolkit 1.7 is a development cycle. 1.6 is the cycle with stable releases. That's why it was set to preferred. We may want to make sure this doesn't turn the install into something with unstable API.

Reply:
Re-added the preferred tag to 1.6 and included a comment explaining the release versions for future contributors.